### ISSUE TITLE
PHP: Wrong more at URL

### ISSUE THREAD

**Issue by bsstoner** [duckduckgo/zeroclickinfo-fathead#425]
More at link goes to freedictionary.com

https://duckduckgo.com/?q=php+date

------
IA Page:  http://duck.co/ia/view/php 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @duckduckgo

**Comment by sahildua2305** [comment]
So it's a case of two IAs being triggered on the query. I am not sure how is such a case handled and how is an IA chosen to display as default.
/cc @moollaza

**Comment by bsstoner** [comment]
@sahildua2305 how do you know that's the issue? As far as I know, we don't have any IA's that point to freedictionary.com.

Is it related to the stray open bracket? That also seems odd:
<img width="877" alt="screen shot 2016-10-06 at 6 31 43 am" src="https://cloud.githubusercontent.com/assets/126358/19149202/aaaf2812-8b8e-11e6-90cf-7a7c1e536ad1.png">

**Comment by sahildua2305** [comment]
@bsstoner I am deducing this from the fact that if you click on `Answer` tab, you will see expected results.

**Comment by bsstoner** [comment]
@sahildua2305 Why is the IA under "Answer" the expected results? I think either IA is an acceptable answer. The fact that the more at link is wrong here in PHP Manual should be completely independent of the other IA.

**Comment by sahildua2305** [comment]
@bsstoner Sorry! Now I get it. I see that currently the PHP Fathead is internal and it will be replaced by a new fathead https://github.com/duckduckgo/zeroclickinfo-fathead/pull/399

**Comment by VitorVRS** [comment]
@bsstoner you can see the new fathead here: https://beta.duckduckgo.com/?q=php+date

**Comment by moollaza** [comment]
@bsstoner that's really odd. I updated the live metadata for the new PHP Fathead to work correctly on Beta. I have no idea though why the More at URL for the live Fathead changed. The URL should be coming straight from the DB.

Oddly we saw the same thing happen for the CSS Fathead when it was first launched: #306 

There must be some internal bug that modifies the More at to point to thefreedictionary.com?

@jdorweiler any ideas what's going on there? I haven't modified the production PHP Fathead files on S3.

Anyway, I will be merging the new Fathead shortly which should fix this.
